tests/scripts/pulp_rpm/test_rpm_sync_publish.sh

```diff
@@ -42,6 +42,14 @@ expect_succ pulp rpm repository update --repository "cli_test_rpm_repository" --
 expect_succ pulp rpm repository show --repository "cli_test_rpm_repository"
 test "$(echo "$OUTPUT" | jq -r '.description')" = "null"
 
+# sqlite metadata removal from pulp_rpm (pulp/pulp_rpm#3328)
+if pulp debug has-plugin --name "rpm" --specifier "<3.25.0"
+then
+  expect_succ pulp rpm repository update --name "cli_test_rpm_repository" --no-sqlite-metadata
+else
+  expect_fail pulp rpm repository update --name "cli_test_rpm_repository" --no-sqlite-metadata
```

Comment:
This is the same command on both branches, is that intentional?  I wouldn't expect that it can both succeed and fail.

Reply:
It succeeds if one is using `pulp_rpm<3.25.0` (which still support sqlite-metadata option) but fails otherwise.

Reply:
I misread it, whoops.  My bad